msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards are known for their ability to adapt to the host application's overall styling. However, their adaptability is limited when it comes to the layout. The static layout approach results in unsatisfactory user experiences, especially across different screen sizes and resolutions.
+
+Adaptive Card responsive layout allows you to design Adaptive Cards that adapt to different card widths. Responsive Adaptive Cards dynamically adjust their layout based on the actual width of the card, ensuring optimal display regardless of the context in which the card is displayed. When you design an Adaptive Card for your app, use the `targetWidth` property to make your card responsive and provide a better experience to your users on Teams mobile or desktop, regardless of the screen size.
+
+#### Configure Adaptive Card responsive layout
+
+The key element to create a responsive Adaptive Card is `CardElement.targetWidth`. The `targetWidth` property allows you to control the visibility of any element by linking it to the width of the card. Add the `targetWidth` property for all elements in an Adaptive Card to define different layouts for different card widths. The value of `targetWidth` can either be `veryNarrow`, `narrow`, `standard`, or, `wide`:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `veryNarrow` | The card width on a meeting chat pane on desktop. |
+  | `narrow` | The card width on a mobile phone in portrait mode. |
+  | `standard` | The card width on a mobile phone in landscape mode, tablet in portrait mode, and chats in desktop. |
+  | `wide` | The card width on a tablet in landscape mode, and chats and channels in desktop. The card width on a chat pane on desktop can vary from `standard` to `wide`. |
+
+  
+
+  The value of `targetWidth` refers to the relative width of the card and allows more flexibility in adapting to different screen sizes. You can decide that certain elements are visible as per the width of the card you're designing. For example, when you design a responsive card, you can decide that certain elements are only visible when the width of the card is very narrow. To do that, you should set `targetWidth`:`veryNarrow` on that specific element.
```

Comment:
This paragraph doesn't read too well IMO. I'd suggest rephrasing as follows:

As you design your card, think about how elements should be optimally laid out for each card width. Then, use the `targetWidth` to make certain elements visible only when the card is a certain width and invisible otherwise. Example:

```json
{
  "type": "AdaptiveCard",
  "version": "1.5",
  "body": [
    {
      "type": "TextBlock",
      "text": "This text is only visible when that card's width is 'veryNarrow', which will happen when the card is viewed in a meeting chat pane"
      "targetWidth": "veryNarrow"
    }
  ]
}
```

Sometimes, you might need to define the `targetWidth` property such that an element is visible is the width of the card is at least or at most a certain value. For that, you can use the `atLeast` and `atMost` modifier prefixes followed by a semicolon and an actual target width. For example:

```json
{
  "type": "AdaptiveCard",
  "version": "1.5",
  "body": [
    {
      "type": "TextBlock",
      "text": "This text is only visible when that card's width is at least 'Narrow', so it will also be visible when the card's width is 'standard' or 'wide', but it will not be visible when the card's width is 'veryNarrow'."
      "targetWidth": "atLeast:Narrow"
    },
    {
      "type": "TextBlock",
      "text": "This text is only visible when that card's width is at most 'Narrow', so it will also be visible when the card's width is 'veryNarrow' but it will not be visible when the card's width is 'standard' or 'wide'."
      "targetWidth": "atMostNarrow"
    }
  ]
}
```